pkg/minikube/image/cache.go

```diff
@@ -74,9 +74,9 @@ func SaveToDir(images []string, cacheDir string, overwrite bool) error {
 			if err := saveToTarFile(image, dst, overwrite); err != nil {
 				if err == errCacheImageDoesntExist {
 					out.WarningT("The image you are trying to add {{.imageName}} doesn't exist!", out.V{"imageName": image})
```

Comment:
I know this isn't related to this PR, but this error message is so grammatically poor. Could you change it something a little better? I would be forever in your debt.

Reply:
I reworded it; no guarantee it's any better.

Reply:
this is much better, thanks